### PR TITLE
Create a way to delete a switch from Kilda cache

### LINKED ISSUE #1074: Create a way to delete a switch from Kilda cache
This must be implemented in Cache topology.
Cache topology must read the CommandMessage from Kafka and delete the switch from CacheBolt.

A part of #328 Create a way to delete switch

### PR DESCRIPTION
**Pull request by Settis**
Closes #1074 issue.

### CODE REVIEW

**Comment by nikitacherevko** [comment]
@Settis Could you please resolve conflicts?

**Comment by abochkarev** [comment]
@nikitacherevko I will resolve conflicts as soon as possible.

**Comment by abochkarev** [comment]
@nikitacherevko I've managed to resolve merge conflicts, but need to test this functionality. In progress...

File: services/wfm/src/main/java/org/openkilda/wfm/topology/cache/service/CacheService.java [hunk 1]
```diff
@@ -400,4 +402,17 @@ public void handleFlowEvent(FlowInfoData flowData, ISender sender, String correl
...
     }
+
+    /**
+     * Deletes a switch from a cache.
+     *
+     * @param request the instance of {@link SwitchDeleteRequest}.
+     * @param sender the instance of {@link Sender}.
+     * @param correlationId the log correlation id.
+     */
+    public void deleteSwitch(SwitchDeleteRequest request, Sender sender, String correlationId) {
```
Comment (abochkarev) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/cache/service/CacheService.java]: Not sure that adding the method 'deleteSwitch' is a good idea. Need to discuss with @Settis

File: services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java [hunk 2]
```diff
@@ -312,7 +312,7 @@ private IslInfoData convertToIslInfoData(LinkDto dto) {
     private SwitchInfoData convertToSwitchInfoData(SwitchDto dto) {
         return new SwitchInfoData(
                 new SwitchId(dto.getSwitchId()),
-                SwitchState.from(dto.getState()),
+                SwitchState.valueOf(dto.getState()),
```
Comment (sergii-iakovenko) [inline services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java]: How do we recoup ability to accept "state" in lower/upper case?

Comment (abochkarev) [reply services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java]: W/o the method "from" it won't be possible. I'm reverting back this change.

Comment (sergii-iakovenko) [reply services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java]: I don't like the method "from" either, that's why we consider migrating to upper-case in enums.

**Pull request merged**